### DISCUSSION TITLE
Version for Ryzen5 5600g

### DISCUSSION THREAD

**Discussion by coimbrox** [ChefKissInc/NootedRed#42] [General]
is there any forecast to release the patch for the 5600g? I installed without having the acceleration and it's very bad for now

**Comment by VisualEhrmanntraut** [comment]
#11, cause is yet to be found

**Comment by VisualEhrmanntraut** [comment]
It is working now if you weren't aware @coimbrox

**Reply by lucasolegario** [reply]
@ChefKissInc I have Ryzen 5 5600g and it had installed with partial acceleration, but I tried to reinstall with the new NootedRed kext and it's happening black screen after apple logo, do you have any idea what it could be? Thank you! 

Congratulations on your excellent work

**Reply by VisualEhrmanntraut** [reply]
Change SMBIOS

**Reply by lucasolegario** [reply]
> Change SMBIOS

I'll try, I'm using SMBIOS from MacPro7,1
